lib/templates.ts

```diff
@@ -1395,6 +1398,71 @@ export async function deleteTemplate(formID: string): Promise<FormRecord | null>
   return _parseTemplate(templateMarkedAsDeleted);
 }
 
+/**
+ * Restores a form template from the archived state.
+ * @param formID ID of the form template
+ * @returns A boolean status if operation is sucessful
+ */
+export async function restoreTemplate(formID: string): Promise<FormRecord | null> {
```

Comment:
Now that we have `restoreTemplate` we should probably rename `deleteForm` (with associated logs/errors...) to `archiveForm`.